### ISSUE TITLE
Integrate dedicated stubs

### ISSUE THREAD

**Issue by pg9182** [R2Northstar/NorthstarLauncher#79]
TLDR: I got the dlls I created for my Docker container working on Windows, drastically reducing the resource usage of the dedicated server by eliminating the need for softwared3d11 (and the multiple gigabytes of memory usage) or a GPU. This will serve as an interim solution until @Mauler125 gets around to the patches for completely eliminating matsys.

[pg9182/northstar-dedicated@6b5645e/src/d3d11](https://github.com/pg9182/northstar-dedicated/tree/6b5645e3bf09fd9d87732ecf52ab57dc632d495e/src/d3d11)
[pg9182/northstar-dedicated@6b5645e/src/gfsdk](https://github.com/pg9182/northstar-dedicated/tree/6b5645e3bf09fd9d87732ecf52ab57dc632d495e/src/gfsdk)
https://github.com/R2Northstar/Northstar/issues/149#issuecomment-1041396597

[Discord > #server-hosting > #windows-stubs : 943573635992547378](https://discord.com/channels/920776187884732556/943369531538173993/943573635992547378)

> so as discussed on VC, I'll put the stubs in a new NorthstarStubs repo with CI and binary releases, then bob'll adopt it into the org and give me admin on that repo, then I'll open a PR to load the stubs in the launcher from bin/x64_dedi (if it exists) for dedi, then I'll open a PR to add the stubs to the release packages

**Comment by pg9182** [comment]
![image](https://user-images.githubusercontent.com/96569817/154452923-95dfd745-a0d9-4fe4-9c9f-4967b53efd0d.png)

**Comment by pg9182** [comment]
Done. Should be in v1.5.1.
